### ISSUE TITLE
Use Surfman with webrender  couldn't render fonts?

### ISSUE THREAD

**Issue by Hankszhang** [servo/surfman#196]
It seems that surfman do not support webrender's font render, I try to impl [Webrender's multiwindow example](https://github.com/servo/webrender/blob/master/examples/multiwindow.rs) with surfman, I did get the rect，but no fonts got rendered.
I did those steps:
```
        let font_key = self.webrender_api.generate_font_key();
        let font_bytes = load_file(font_path);

        let mut txn = Transaction::new();
        txn.add_raw_font(font_key, font_bytes, 0);

        let font_instance_key = self.webrender_api.generate_font_instance_key();
        txn.add_font_instance(font_instance_key, font_key, glyph_size, None, None, Vec::new());

        self.webrender_api.send_transaction(self.document_id, txn);
```

and then put my glyphs  into display_list through `builder.push_text`, but no text are rendered, did I miss some config?

**Comment by jdm** [comment]
Are you using macOS, by any chance? If so, try adding the `allow_texture_swizzling: true` option to webrender::RendererOptions. There is a known issue on certain macOS graphics drivers that makes fonts not appear.

**Comment by jdm** [comment]
Sorry, I meant `allow_texture_swizzling: false`, since the default value is true.

**Comment by Hankszhang** [comment]
Yes，I'm using MacOS
I found the root cause is the version of `gleam` is too low（0.6.9），the issue resolved after update the version to 0.12
